src/util/sharing_map.h

```diff
@@ -450,7 +450,7 @@ SHARING_MAPT2(, size_type)::erase(
     return 0;
 
   node_type *del=nullptr;
-  unsigned del_bit;
+  unsigned del_bit=0;
```

Comment:
I'm pretty sure `unsigned` should not be used here at all and any such use should be a `std::size_t` instead.